src/less/expressive-loader/expressive-loader.less

```diff
@@ -14,8 +28,8 @@
     gap: var(--spacing-300);
     justify-content: flex-end;
     margin: 0 auto;
-    max-width: 100%;
-    width: 375px;
+    max-width: 375px;
```

Comment:
Good catch on max-width and width, @ArtBlue. I've switched them here so width is 100%.